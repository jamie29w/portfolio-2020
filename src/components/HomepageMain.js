import React from 'react';
import { css } from '@emotion/core';
import styled from '@emotion/styled';
import { Link } from 'gatsby';

import DarkModeSwitch from './DarkModeSwitch';
import Footer from './Footer';

const HomepageMain = () => {
  return (
    <HomepageMainWrapper>
      <NameWrapper>
        <h2>Jamie</h2>
        <h2>Woodmancy</h2>
      </NameWrapper>
      <DarkModeSwitchWrapper>
        <DarkModeSwitch />
      </DarkModeSwitchWrapper>
      <TitleBlock>
        <h1>Front</h1>
        <h1>End</h1>
        <h1>Engineer</h1>
      </TitleBlock>
      <AboutBlock>
        <p>Balanced form and function</p>
        <p>[Under construction]</p>
      </AboutBlock>
      <CtaWrapper>
        <Link className='btn' to='/work/'>
          See my work
        </Link>
      </CtaWrapper>
      <Footer
        css={css`
          grid-area: footer;
        `}
      />
    </HomepageMainWrapper>
  );
};

export default HomepageMain;

const HomepageMainWrapper = styled.div`
  display: grid;
  grid-template-areas:
    'nameWrapper'
    'titleBlock'
    'aboutBlock'
    'cta'
    'footer';
  grid-template-columns: 1fr;
  grid-template-rows: auto 1.5fr 1fr 1fr auto;
  min-height: 100vh;
  padding: var(--gutterHorizontal);
  position: relative;
`;

const NameWrapper = styled.div`
  align-items: flex-start;
  grid-area: nameWrapper;

  & > h2 {
    margin-top: 0;
  }

  @media (min-width: 48em) {
    display: flex;
    flex-direction: row;
    justify-content: center;
    width: 100%;

    & > h2 {
      margin-left: calc(var(--spacing) / 2);
    }
  }
`;

const DarkModeSwitchWrapper = styled.div`
  align-items: center;
  position: absolute;
  right: calc(0.5em + var(--gutterHorizontal));
  top: calc(0.5em + var(--gutterHorizontal));
`;

const TitleBlock = styled.div`
  display: flex;
  flex-direction: column;
  grid-area: titleBlock;
  justify-content: flex-end;

  & > h1 {
    margin-top: 0;
  }

  @media (min-width: 48em) {
    flex-direction: row;
    align-items: flex-end;
    justify-content: center;

    & > h1:not(:first-of-type) {
      margin-left: calc(var(--spacing) / 2);
    }
  }
`;

const AboutBlock = styled.div`
  display: flex;
  flex-direction: column;
  grid-area: aboutBlock;
  margin-top: var(--spacing);
  justify-content: flex-start;

  & > p {
    margin-top: 0;
  }

  @media (min-width: 48em) {
    align-items: center;
  }
`;

const CtaWrapper = styled.div`
  align-items: center;
  display: flex;
  grid-area: cta;
  justify-content: center;

  & > a {
    display: flex;
    border: 2px solid var(--primary);
    flex: 1;
    justify-content: center;
    margin-top: 0;
    padding: calc(0.25 * var(--spacing)) 0;
    transition: background-size 0.2s ease-in, border 0.2s ease-in;

    &:before {
      background: transparent;
      height: 100%;
      border-top: 2px solid var(--secondary);
      border-bottom: 2px solid var(--secondary);
    }
    &:after {
      background: transparent;
      bottom: 0;
      content: '';
      left: 0;
      height: 100%;
      border-left: 2px solid var(--secondary);
      border-right: 2px solid var(--secondary);
      position: absolute;
      transform: scaleY(0);
      transition: all 0.2s ease-in;
      width: 100%;
    }

    &:focus,
    &:hover {
      background-size: 100% 100%;
      border: 2px solid transparent;
      transition: background-size 0.2s ease-out, border 0.2s ease-out;

      &:after {
        transform: scaleY(1);
        transition: all 0.2s ease-out;
      }
    }

    @media (min-width: 48em) {
      max-width: 200px;
    }
  }
`;
