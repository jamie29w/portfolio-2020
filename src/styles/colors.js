import Color from 'color';

export const primary_root = Color('#36B7FC');
export const secondary_root = Color('#FF9F2C');

export const primaryColor = Color(primary_root);
export const secondaryColor = Color(secondary_root);

export const colors = {
  light: {
    primary: primaryColor,
    primaryHover: primaryColor.lighten(0.3),
    secondary: secondaryColor,
    secondaryHover: primaryColor.lighten(0.3),
    header: 'rgba(35, 35, 35, 0.87)',
    paragraph: 'rgba(75, 75, 75, 0.87)',
    background: 'rgba(255, 255, 255, 1)',
    boxShadowColor: 'rgba(0, 0, 0, 0.25)',
    divider: 'rgba(255, 255, 255, 0.12)',
    gradient1HomePage: 'rgba(255, 255, 255, 1)',
    gradient2HomePage: primaryColor.lighten(0.43),
    navFooterBackground: 'rgba(255, 255, 255, 1)',
  },
  dark: {
    primary: primaryColor.lighten(0.3).fade(0.13),
    primaryHover: primaryColor.lighten(0.6).fade(0.13),
    secondary: secondaryColor.lighten(0.1).fade(0.13),
    secondaryHover: secondaryColor.lighten(0.4).fade(0.13),
    header: 'rgba(225, 225, 225, 0.87)',
    paragraph: 'rgba(200, 200, 200, 0.87)',
    background: 'rgba(0, 0, 0, 0.87)',

    boxShadowColor: 'rgba(255, 255, 255, 0.35)',
    divider: 'rgba(255, 255, 255, 0.82)',
    gradient1HomePage: 'rgba(33, 33, 33, 1)',
    gradient2HomePage: 'rgba(0, 0, 0, 0.87)',
    navFooterBackground: 'rgba( 4, 4, 4, 1)',
  },
};
