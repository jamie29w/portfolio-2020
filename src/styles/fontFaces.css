/* roboto font-face declarations */
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 900;
  src: url('../../assets/fonts/Roboto/Roboto-Black.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: italic;
  font-weight: 900;
  src: url('../../assets/fonts/Roboto/Roboto-BlackItalic.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 700;
  src: url('../../assets/fonts/Roboto/Roboto-Bold.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: italic;
  font-weight: 700;
  src: url('../../assets/fonts/Roboto/Roboto-BoldItalic.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 500;
  src: url('../../assets/fonts/Roboto/Roboto-Medium.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: italic;
  font-weight: 500;
  src: url('../../assets/fonts/Roboto/Roboto-MediumItalic.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 400;
  src: url('../../assets/fonts/Roboto/Roboto-Regular.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: italic;
  font-weight: 400;
  src: url('../../assets/fonts/Roboto/Roboto-Italic.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: url('../../assets/fonts/Roboto/Roboto-Light.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: italic;
  font-weight: 300;
  src: url('../../assets/fonts/Roboto/Roboto-LightItalic.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 100;
  src: url('../../assets/fonts/Roboto/Roboto-Thin.ttf');
}
@font-face {
  font-family: 'Roboto';
  font-style: italic;
  font-weight: 100;
  src: url('../../assets/fonts/Roboto/Roboto-ThinItalic.ttf');
}

/* open sans font-face declarations */
@font-face {
  font-family: 'Open Sans';
  font-style: normal;
  font-weight: 800;
  src: url('../../assets/fonts/Open_Sans/OpenSans-ExtraBold.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: italic;
  font-weight: 800;
  src: url('../../assets/fonts/Open_Sans/OpenSans-ExtraBoldItalic.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: normal;
  font-weight: 700;
  src: url('../../assets/fonts/Open_Sans/OpenSans-Bold.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: italic;
  font-weight: 700;
  src: url('../../assets/fonts/Open_Sans/OpenSans-BoldItalic.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: normal;
  font-weight: 600;
  src: url('../../assets/fonts/Open_Sans/OpenSans-SemiBold.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: italic;
  font-weight: 600;
  src: url('../../assets/fonts/Open_Sans/OpenSans-SemiBoldItalic.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: normal;
  font-weight: 400;
  src: url('../../assets/fonts/Open_Sans/OpenSans-Regular.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: italic;
  font-weight: 400;
  src: url('../../assets/fonts/Open_Sans/OpenSans-Italic.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: normal;
  font-weight: 300;
  src: url('../../assets/fonts/Open_Sans/OpenSans-Light.ttf');
}
@font-face {
  font-family: 'Open Sans';
  font-style: italic;
  font-weight: 300;
  src: url('../../assets/fonts/Open_Sans/OpenSans-LightItalic.ttf');
}

/* raleway font-face declarations */
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 900;
  src: url('../../assets/fonts/Raleway/Raleway-Black.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 900;
  src: url('../../assets/fonts/Raleway/Raleway-BlackItalic.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 800;
  src: url('../../assets/fonts/Raleway/Raleway-Black.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 800;
  src: url('../../assets/fonts/Raleway/Raleway-BlackItalic.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 700;
  src: url('../../assets/fonts/Raleway/Raleway-Bold.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 700;
  src: url('../../assets/fonts/Raleway/Raleway-BoldItalic.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 600;
  src: url('../../assets/fonts/Raleway/Raleway-Bold.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 600;
  src: url('../../assets/fonts/Raleway/Raleway-BoldItalic.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 500;
  src: url('../../assets/fonts/Raleway/Raleway-Medium.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 500;
  src: url('../../assets/fonts/Raleway/Raleway-MediumItalic.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 400;
  src: url('../../assets/fonts/Raleway/Raleway-Regular.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 400;
  src: url('../../assets/fonts/Raleway/Raleway-Italic.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 300;
  src: url('../../assets/fonts/Raleway/Raleway-Light.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 300;
  src: url('../../assets/fonts/Raleway/Raleway-LightItalic.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 200;
  src: url('../../assets/fonts/Raleway/Raleway-Light.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 200;
  src: url('../../assets/fonts/Raleway/Raleway-LightItalic.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: normal;
  font-weight: 100;
  src: url('../../assets/fonts/Raleway/Raleway-Thin.ttf');
}
@font-face {
  font-family: 'Raleway';
  font-style: italic;
  font-weight: 100;
  src: url('../../assets/fonts/Raleway/Raleway-ThinItalic.ttf');
}
