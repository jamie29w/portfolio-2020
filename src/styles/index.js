export { GlobalComponent } from './GlobalComponent';
export { default, theme } from './theme';
