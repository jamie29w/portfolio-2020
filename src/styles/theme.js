export const theme = {
  colors: {
    primary: '#24b47e;',
    accentLight: '#05d5ff',
    accentMid: '#3297d3;',
    background: '#fff',
    bodyText: '#525f7f',
  },
  spacing: 1,
};

export default function createTheme() {
  return { palette: createPalette('light'), spacing: 1 };
}

const common = {
  black: 'rgba(0, 0, 0, 1.0)',
  white: 'rgba(255, 255, 255, 1.0)',
};

const light = {
  text: {
    primary: 'rgba(0, 0, 0, 0.87)',
    secondary: 'rgba(0, 0, 0, 0.54)',
    disabled: 'rgba(0, 0, 0, 0.38)',
    hint: 'rgba(0, 0, 0, 0.38)',
  },
  divider: 'rgba(0, 0, 0, 0.12)',
  background: {
    primary: common.white,
    secondary: 'rgba(0, 0, 0, 0.87)',
  },
};

const dark = {
  text: {
    primary: common.white,
    secondary: 'rgba(255, 255, 255, 0.7)',
    disabled: 'rgba(255, 255, 255, 0.5)',
    hint: 'rgba(255, 255, 255, 0.5)',
  },
  divider: 'rgba(255, 255, 255, 0.12)',
  background: {
    primary: 'rgba(0, 0, 0, 0.87)',
    secondary: common.white,
  },
};

const types = { light, dark };

function createPalette(type) {
  const primary = {
      light: '#CE7870',
      dark: '#CE7870',
    },
    secondary = {
      light: '#BECBCF',
      dark: '#BECBCF',
    },
    tertiary = {
      light: '#ECBCB0',
      dark: '#ECBCB0',
    },
    success = {
      light: '#6E7D75',
      dark: '#6E7D75',
    },
    warning = {
      light: '#EBA966',
      dark: '#EBA966',
    };

  const paletteOutput = {
    common,
    type,
    primary: primary[type],
    secondary: secondary[type],
    tertiary: tertiary[type],
    warning: warning[type],
    success: success[type],
    ...types[type],
  };

  return paletteOutput;
}
