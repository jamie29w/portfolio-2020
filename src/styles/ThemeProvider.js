import React, { createContext, useEffect, useState } from 'react';
import { colors } from './colors';

export const ThemeContext = createContext(null);

export const ThemeProvider = ({ children }) => {
  const [themeType, rawSetThemeType] = useState(undefined);

  // reading on render prevents toggle from getting out of sync
  // after MagicScriptTag does its job
  useEffect(() => {
    const root = window.document.documentElement;
    const initialColorValue = root.style.getPropertyValue(
      '--initial-color-mode'
    );

    rawSetThemeType(initialColorValue);
  }, []);

  function setThemeType(newValue) {
    const root = window.document.documentElement;

    rawSetThemeType(newValue);
    window.localStorage.setItem('color-mode', newValue);

    root.style.setProperty(
      '--primary',
      newValue === 'light' ? colors.light.primary : colors.dark.primary
    );

    root.style.setProperty(
      '--primaryActive',
      newValue === 'light'
        ? colors.light.primaryActive
        : colors.dark.primaryActive
    );

    root.style.setProperty(
      '--secondary',
      newValue === 'light' ? colors.light.secondary : colors.dark.secondary
    );

    root.style.setProperty(
      '--secondaryActive',
      newValue === 'light'
        ? colors.light.secondaryActive
        : colors.dark.secondaryActive
    );

    root.style.setProperty(
      '--headerColor',
      newValue === 'light' ? colors.light.header : colors.dark.header
    );

    root.style.setProperty(
      '--paragraphColor',
      newValue === 'light' ? colors.light.paragraph : colors.dark.paragraph
    );

    root.style.setProperty(
      '--divider',
      newValue === 'light' ? colors.light.divider : colors.dark.divider
    );

    root.style.setProperty(
      '--background',
      newValue === 'light' ? colors.light.background : colors.dark.background
    );

    root.style.setProperty(
      '--gradient1HomePage',
      newValue === 'light'
        ? colors.light.gradient1HomePage
        : colors.dark.gradient1HomePage
    );

    root.style.setProperty(
      '--gradient2HomePage',
      newValue === 'light'
        ? colors.light.gradient2HomePage
        : colors.dark.gradient2HomePage
    );

    root.style.setProperty(
      '--darkModeSwitchHover',
      newValue === 'light' ? colors.light.primary : colors.dark.secondary
    );

    root.style.setProperty(
      '--boxShadowColor',
      newValue === 'light'
        ? colors.light.boxShadowColor
        : colors.dark.boxShadowColor
    );

    root.style.setProperty(
      '--navFooterBackground',
      newValue === 'light'
        ? colors.light.navFooterBackground
        : colors.dark.navFooterBackground
    );
  }

  return (
    <ThemeContext.Provider
      value={{
        themeType,
        setThemeType,
      }}
    >
      {children}
    </ThemeContext.Provider>
  );
};
