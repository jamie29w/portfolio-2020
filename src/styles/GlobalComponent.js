import React, { useContext } from 'react';
import { Global, css } from '@emotion/core';
import emotionNormalize from 'emotion-normalize';
import Color from 'color';

import { ThemeContext } from '../providers/ThemeProvider';
import '../styles/fontFaces.css';

// This component provides the global styles reset and
// base styles for the app.
// GlobalComponent is a single source of styling for the
// gatsby-browser.js and gatsby-ssr.js files which both
// act as the root of the project.

export const GlobalComponent = () => {
  const {
    palette: {
      background: { primary: bgPrimary },
      primary,
      secondary,
      success,
      text: { primary: bodyText },
    },
    spacing,
  } = useContext(ThemeContext);

  const Secondary = Color(secondary);

  return (
    <>
      <Global
        styles={css`
          ${emotionNormalize}
          html {
            background: ${bgPrimary};
            box-sizing: border-box;
            font-family: sans-serif;
            margin: 0;
            padding: 0;
            -ms-text-size-adjust: 100%;
            -webkit-text-size-adjust: 100%;
          }

          *,
          *:before,
          *:after {
            box-sizing: inherit;
          }

          body {
            background: ${bgPrimary};
            color: ${bodyText};
            font-family: 'Open Sans', sans-serif;
            font-weight: 400;
            margin: 0;
            padding: 0;
            -webkit-font-smoothing: antialiased;
            -moz-osx-font-smoothing: grayscale;
          }

          code {
            font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
              monospace;
          }

          h1,
          h2,
          h3,
          h4,
          h5 {
            color: ${secondary};
            font-family: 'Raleway', sans-serif;
            font-weight: 600;
            padding: 0;
            margin: ${spacing}rem 0 0;
          }

          h1 {
            color: ${primary};
            font-size: 3.052rem;
            letter-spacing: -1.5px;
          }

          h2 {
            color: ${success};
            font-size: 2.441rem;
            letter-spacing: -0.5px;
          }

          h3 {
            font-size: 1.953rem;
            letter-spacing: 0px;
          }

          h4 {
            font-size: 1.563rem;
            letter-spacing: 0.25px;
          }

          h5 {
            font-size: 1.25rem;
            letter-spacing: 0px;
          }

          a,
          p,
          ul,
          li {
            font-family: 'Open Sans', sans-serif;
            font-size: 1rem;
            font-weight: 400;
            letter-spacing: 0.5px;
            padding: 0;
            margin: ${spacing}rem 0 0;
          }

          .caption {
            font-size: 0.8rem;
            letter-spacing: 0.4px;
          }

          a {
            color: ${secondary};
            text-decoration: none;

            &:hover,
            &:focus {
              color: ${Secondary.darken(0.5)};
            }
          }

          ul {
            padding-left: 0;
            margin-top: 0;

            & > li {
              list-style: none;
            }
          }
        `}
      />
    </>
  );
};
