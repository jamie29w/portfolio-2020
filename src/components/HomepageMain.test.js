import React from 'react';
import renderer from 'react-test-renderer';

import HomepageMain from './HomepageMain';
import { ThemeProvider } from '../styles';

describe('HomepageMain', () => {
  test.each([['light'], ['dark']])(
    'renders correctly in %s mode',
    themeValue => {
      const comp = renderer
        .create(
          <ThemeProvider value={themeValue}>
            <HomepageMain />
          </ThemeProvider>
        )
        .toJSON();

      expect(comp).toMatchSnapshot();
    }
  );
});
