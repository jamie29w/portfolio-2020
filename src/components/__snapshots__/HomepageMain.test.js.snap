// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageMain renders correctly in dark mode 1`] = `
<div
  className="css-1jrakj0"
>
  <div
    className="css-41gcnh"
  >
    <h2>
      Jamie
    </h2>
    <h2>
      Woodmancy
    </h2>
  </div>
  <div
    className="css-1b2m9z"
  >
    <button
      className="css-9xbtd6"
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M9 2c-1.05 0-2.05.16-3 .46 4.06 1.27 7 5.06 7 9.54 0 4.48-2.94 8.27-7 9.54.95.3 1.95.46 3 .46 5.52 0 10-4.48 10-10S14.52 2 9 2z"
        />
      </svg>
    </button>
  </div>
  <div
    className="css-17s6pf5"
  >
    <h1>
      Front
    </h1>
    <h1>
      End
    </h1>
    <h1>
      Engineer
    </h1>
  </div>
  <div
    className="css-1ah2vuk"
  >
    <p>
      Balanced form and function
    </p>
    <p>
      [Under construction]
    </p>
  </div>
  <div
    className="css-1bmuida"
  >
    <a
      className="btn"
      href="/work/"
    >
      See my work
    </a>
  </div>
  <footer
    className="css-9zmaun"
  >
    <span
      className="caption"
    >
      Woodmancy Dev™ 
      2020
      , built with
    </span>
    <span
      className="css-1g2vfi9"
    >
      <a
        href="https://www.reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="link to react homepage"
          css={
            Object {
              "map": undefined,
              "name": "rmhaug",
              "next": undefined,
              "styles": "
                  height: 20px;
                ",
              "toString": [Function],
            }
          }
          src="test-file-stub"
        />
      </a>
      <a
        href="https://www.gatsbyjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="link to gatsby homepage"
          css={
            Object {
              "map": undefined,
              "name": "rmhaug",
              "next": undefined,
              "styles": "
                  height: 20px;
                ",
              "toString": [Function],
            }
          }
          src="test-file-stub"
        />
      </a>
      <a
        href="https://www.netlify.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="link to netlify homepage"
          css={
            Object {
              "map": undefined,
              "name": "rmhaug",
              "next": undefined,
              "styles": "
                  height: 20px;
                ",
              "toString": [Function],
            }
          }
          src="test-file-stub"
        />
      </a>
    </span>
  </footer>
</div>
`;

exports[`HomepageMain renders correctly in light mode 1`] = `
<div
  className="css-1jrakj0"
>
  <div
    className="css-41gcnh"
  >
    <h2>
      Jamie
    </h2>
    <h2>
      Woodmancy
    </h2>
  </div>
  <div
    className="css-1b2m9z"
  >
    <button
      className="css-9xbtd6"
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M9 2c-1.05 0-2.05.16-3 .46 4.06 1.27 7 5.06 7 9.54 0 4.48-2.94 8.27-7 9.54.95.3 1.95.46 3 .46 5.52 0 10-4.48 10-10S14.52 2 9 2z"
        />
      </svg>
    </button>
  </div>
  <div
    className="css-17s6pf5"
  >
    <h1>
      Front
    </h1>
    <h1>
      End
    </h1>
    <h1>
      Engineer
    </h1>
  </div>
  <div
    className="css-1ah2vuk"
  >
    <p>
      Balanced form and function
    </p>
    <p>
      [Under construction]
    </p>
  </div>
  <div
    className="css-1bmuida"
  >
    <a
      className="btn"
      href="/work/"
    >
      See my work
    </a>
  </div>
  <footer
    className="css-9zmaun"
  >
    <span
      className="caption"
    >
      Woodmancy Dev™ 
      2020
      , built with
    </span>
    <span
      className="css-1g2vfi9"
    >
      <a
        href="https://www.reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="link to react homepage"
          css={
            Object {
              "map": undefined,
              "name": "rmhaug",
              "next": undefined,
              "styles": "
                  height: 20px;
                ",
              "toString": [Function],
            }
          }
          src="test-file-stub"
        />
      </a>
      <a
        href="https://www.gatsbyjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="link to gatsby homepage"
          css={
            Object {
              "map": undefined,
              "name": "rmhaug",
              "next": undefined,
              "styles": "
                  height: 20px;
                ",
              "toString": [Function],
            }
          }
          src="test-file-stub"
        />
      </a>
      <a
        href="https://www.netlify.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="link to netlify homepage"
          css={
            Object {
              "map": undefined,
              "name": "rmhaug",
              "next": undefined,
              "styles": "
                  height: 20px;
                ",
              "toString": [Function],
            }
          }
          src="test-file-stub"
        />
      </a>
    </span>
  </footer>
</div>
`;
